README.md

```diff
@@ -55,6 +55,7 @@ For `around_start_chains` callback `options` hash contains three keys:
   job_class: CheckUsersActivityJob,
   array_of_job_arguments: [1, 2 ,3],
   parallelism: 2,
+  args: [{"job_argument" => "value"}],
```

Comment:
Right before this Ruby code block, it says "hash contains **three** keys", which is now outdated.